landing-page.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="darkreader" content="dark" />
+        <title>namaka</title>
+        <!-- <link rel="stylesheet" href="/demo.css" /> -->
+        <link rel="stylesheet" href="/global.css" />
+
+        <script defer type="importmap-shim" src="/x/importmap.json"></script>
+        <script defer type="module-shim" src="/x/mock.js"></script>
+        <script defer src="/node_modules/es-module-shims/dist/es-module-shims.js"></script>
+    </head>
+    <body>
```

Comment:
whitespace is a broken mix of tabs and spaces here.

even after i install a typescript linter, it wouldn't catch html files like this.

perhaps set the vscode "render whitespace" setting to `boundary`.